docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/ncids_trans.theme

```diff
@@ -68,11 +68,18 @@ function ncids_trans_theme_suggestions_page_alter(array &$suggestions, array $va
 function ncids_trans_theme_suggestions_node_alter(array &$suggestions, array $variables) {
   $node = $variables['elements']['#node'];
 
-  if ($node->bundle() !== "cgov_home_landing" && $node->bundle() !== "cgov_mini_landing") {
+  if ($node->bundle() !== "cgov_home_landing" && $node->bundle() !== "cgov_mini_landing" && $node->bundle() !== "cgov_application_page") {
```

Comment:
You 100% should have to update https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/modules/custom/cgov_application_page/cgov_application_page.module#L104. 

If you *really* need to have a node template, which you probably do to get rid of the `<article>` and other stuff, then you should add the theme suggestion hook to `cgov_application_page.module`.

Reply:
(Yes, the code here probably should have been added to `cgov_home_landing.module` ...)